core/email_api.php

```diff
@@ -1555,3 +1555,29 @@ function email_shutdown_function() {
 		email_send_all();
 	}
 }
+
+/**
+ * Get the list of supported email actions.
+ *
+ * @return array List of actions
+ */
+function email_get_actions() {
+	$t_actions = array( 'updated', 'owner', 'reopened', 'deleted', 'bugnote' );
+
+	if( config_get( 'enable_sponsorship' ) == ON ) {
+		$t_actions[] = 'sponsor';
+	}
+
+	$t_actions[] = 'relation';
```

Comment:
Nope, this was the case for the original code, I'll combine it with the initial array initialization.